### PR TITLE
LINK-1808 | Command for removing expired admin permissions

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
Command for removing expired admin and registration user access permissions. Env variables are also added to allow controlling the expiration thresholds.
### Closes
[LINK-1808](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1808)

[LINK-1808]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1808?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: events/management/commands/remove_unused_permissions.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            registration_admin_organizations__isnull=False,
+        ):
+            user.registration_admin_organizations.clear()
+            admins_count += 1
+
+        return admins_count
+
+    @staticmethod
+    def _handle_registration_users(datetime_now):
+        admins_count = 0
```
Comment (charn) [inline events/management/commands/remove_unused_permissions.py:62]: This could be renamed since these are not admins.

Comment (harriris-vincit) [reply events/management/commands/remove_unused_permissions.py]: Changed as suggested. Also mentioned registration users separately in the command's help text.

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr924.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by jorilindell — APPROVED** [review]
Looks very good. LGTM

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr924.api.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**